app/controllers/skills/new.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.ObjectController.extend({
+  needs: ['skills'],
+  // replace this with data from rails api call (get all groups)
+  groups: ["1407", "1409", "1410", "1412"],
+  featured: ['True', 'False'],
+
+  actions: {
+    saveSkill: function () {
+      this.store.createRecord('skill', {
+        name: this.get('name'),
+        group: this.get('group'),
+        featured: this.get('featured'),
+      }).save().then(function () {
```

Comment:
Also, the validations ideally should be done on both the ember side and the rails side.  There is some good validation info here: http://ember.vicramon.com/creating-a-lead